client/post-editor/editor-drawer/copy-post.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import PostSelector from 'my-sites/post-selector';
+
+class EditorDrawerCopyPost extends Component {
+
+	static propTypes = {
+		site: PropTypes.object.isRequired,
```

Comment:
If possible, try to pass these props via `react-redux` `connect`. These state selectors may be of use:

- [`getSelectedSiteId`](https://github.com/Automattic/wp-calypso/blob/37d3a88e3732ec131ad518eb0266ed076855e927/client/state/ui/selectors.js#L28-L36)
- [`getEditorPostId`](https://github.com/Automattic/wp-calypso/blob/37d3a88e3732ec131ad518eb0266ed076855e927/client/state/ui/editor/selectors.js#L12-L20)